packages/web-frontend/src/containers/MeasureBar/Control.js

```diff
@@ -110,6 +110,7 @@ export const Control = ({
   emptyMessage,
   selectedMeasure,
   defaultDates,
+  limits,
```

Comment:
```suggestion
  datePickerLimits,
```
In the context of the measure bar control, I'd assume limits referred to constraints on the data itself

Reply:
Renamed